cli/integrity/integrity.go

```diff
@@ -0,0 +1,31 @@
+package integrity
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// Signer implements high-level API for operations that require
+// a private key.
+type Signer interface {
+	// AddIntegrityHashes generates files with hashes for integrity
+	// checks and signs them.
+	Sign(appList []util.AppListEntry, baseDir string) error
+}
+
+// dummySigner is a noop integrity check preparator.
+type dummySigner struct{}
+
+// Sign is a noop preparation of integrity check files.
+func (dummySigner) Sign([]util.AppListEntry, string) error {
```

Comment:
```suggestion
// Sign is a noop preparation of integrity check files.
func (dummySigner) Sign(basePath string, apps []util.AppListEntry) error {
```
where -> applist seems less confusing than applist -> where.
Please, also name arguments in the interface.

Reply:
Fixed